Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -363,8 +365,49 @@
     m_scrollContainerLayer->setEventRegion(WTFMove(eventRegion));
 }
 
-bool UnifiedPDFPlugin::handleMouseEvent(const WebMouseEvent&)
+enum class AltKeyIsActive : bool { No, Yes };
+
+static WebCore::Cursor::Type toWebCoreCursorType(UnifiedPDFPlugin::ElementsUnderMouse elementsUnderMouse, AltKeyIsActive altKeyIsActive = AltKeyIsActive::No)
+{
+    auto cursor = WebCore::Cursor::Type::Pointer;
+
+    if (elementsUnderMouse.containsAny({ UnifiedPDFPlugin::ElementUnderMouse::Link, UnifiedPDFPlugin::ElementUnderMouse::Control, UnifiedPDFPlugin::ElementUnderMouse::Icon }))
```

Comment:
this could probably be just a bunch of early returns instead of an if-else chain, right?

Reply:
Sure, yeah, I think I can re-write like this

```cpp
static WebCore::Cursor::Type toWebCoreCursorType(UnifiedPDFPlugin::ElementsUnderMouse elementsUnderMouse, AltKeyIsActive altKeyIsActive = AltKeyIsActive::No)
{
    if (elementsUnderMouse.containsAny({ UnifiedPDFPlugin::ElementUnderMouse::Link, UnifiedPDFPlugin::ElementUnderMouse::Control, UnifiedPDFPlugin::ElementUnderMouse::Icon }))
        return WebCore::Cursor::Type::Hand;
    if (altKeyIsActive == AltKeyIsActive::Yes)
        return WebCore::Cursor::Type::Hand;
    if (elementsUnderMouse.containsAny({ UnifiedPDFPlugin::ElementUnderMouse::Text, UnifiedPDFPlugin::ElementUnderMouse::TextField }))
        return WebCore::Cursor::Type::IBeam;

    return WebCore::Cursor::Type::Pointer;
}
```